supporting_content/dimensions/listunitprice.md

```diff
@@ -0,0 +1,83 @@
+# Column: ListUnitPrice
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR;<br>AWS Price List API   | pricing/publicOnDemandCost: The total cost for the line item based on public On-Demand Instance rates. If you have SKUs with multiple On-Demand public costs, the equivalent cost for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>Since the pricing/publicOnDemandCost currently doesn't provide the correct price in all tier-based pricing use cases, we need to rely on the AWS Price List API. |
+| GCP       | BigQuery Pricing Data Export | Not available in BigQuery Billing Export, but can be resolved from the list_price Struct in the BigQuery Pricing Data Export |
+| Microsoft | Cost details;<br>Azure Retail Prices REST API | pay-as-you-goPrice/payGPrice: Retail price for the resource<br>Note: Identified some discrepancy between the value in Cost details and the Retail Price obtained from the Azure Retail Prices REST API, particularly in the context of CSPs. Might be related to Partner Billing UC? |
+| OCI       | List Pricing REST API        | Not available in Cost and Usage Report but can be resolved from the List Pricing REST API |
+
+## Example scenarios for current provider data
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Scenario                                 | Pattern                   |
+|:----------|:-----------------------------------------|:----------------------------|
+| AWS       | Flat-rate and tier-based list unit price | See [Current ListUnitPrice UCs and Data Samples - AWS sheet](https://docs.google.com/spreadsheets/d/1TpoXiu3aW_ENXvbONsjb6S1fqC7tJ8RcBNtFbsS9Qjw/edit#gid=726782282) for current data samples, as available in CUR files and AWS Price List API |
+| GCP       | Flat-rate and tier-based list unit price | See [Current ListUnitPrice UCs and Data Samples - GCP sheet](https://docs.google.com/spreadsheets/d/1TpoXiu3aW_ENXvbONsjb6S1fqC7tJ8RcBNtFbsS9Qjw/edit#gid=318165912) for current data samples, as available in BigQuery Pricing Data Export |
+| Microsoft | Flat-rate and tier-based list unit price | See [Current ListUnitPrice UCs and Data Samples - Microsoft sheet](https://docs.google.com/spreadsheets/d/1TpoXiu3aW_ENXvbONsjb6S1fqC7tJ8RcBNtFbsS9Qjw/edit#gid=1938507497) for current data samples, as available in Cost details and Azure Retail Prices REST API |
```

Comment:
We should either move the data to this repo or open this spreadsheet for anonymous access. Doesn't make sense to link to a doc as part of a spec that nobody has access to.

Reply:
Crucial content has been copied to MD files. 

**Notes:** 

- The link to one of the docs is still present for additional information, but since the main content crucial for ListUnitPrice in the billing context is available within the supporting materials, **we can omit the link.** 
- We should decide **how to handle any significant spreadsheets** when/if we decide to prepare an **appendix for Pricing support scenarios**.

Reply:
Discussed at maintainers' meeting. For the time being, links to spreadsheets within the supporting content are acceptable, even if those are not available to the public.